### PR TITLE
Adding docs about robot users

### LINKED ISSUE #248: Custom user is not able to push an image
I would like to have my CI/CD pipeline push singularity images to my sregistry instance. In order to do that, I needed to create a custom sregistry user for my CI/CD, because obviously I do not want to use the token of any other regular user for that. I used the Django Administration site to add this user and a token for this user was automatically created.

When I tried to push an image with this custom user, I noticed that I am not able to do so because a push is only allowed if a collection already exists and is owned by this user. Since I cannot login on the sregistry website with this custom user I had to find another way to create a collection. So I created a team and team-owned collection in the web site using my own user account. Then I added the custom user to that team using the `python manage.py shell` and then making the changes manually, because I can only invite users in the web interface (I cannot add them).

When I setup the `remotes.yaml` and `.sregistry` for the custom user accordingly, I still get `FATAL:   Unable to push image to library: request did not succeed: http status code: 500`. which I do not get for the regular account.

Any ideas what would cause this? Is there more I need to do besides adding the custom user to the team?

I am using commit 1dc800b8eff66013df59a80c1801847874e11341 of sregistry and singularity v3.3.0 (installed from source) on ubuntu 18.04.

### PR DESCRIPTION
**Pull request by vsoch**
This is a WIP PR to add support (and instructions) for a basic robot user. It will close #248 

Signed-off-by: Vanessa Sochat <vsochat@stanford.edu>

### CODE REVIEW

File: docs/_docs/accounts/robots.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+ 2. Since the collection must already exist and you cannot log in as the robot user, you should create the collection with your user account, create a [team]({{ site.url }}{{ site.baseurl }}/docs/setup/teams) and add the collection to be owned by the team, and then add the robot user to the team inside the container like so:
+
+First, enter the uwsgi container (the name of your container may be different)
+
+```
+docker exec -it sregistry_uwsgi_1 bash
+python manage.py shell
+```
+
+Find your robot user, and your team (replace `gitlab-ci` with the name of your user)
```
Comment (RonaldEnsing) [inline docs/_docs/accounts/robots.md]: Maybe just replace `gitlab-ci` with `myuser` (similar to `myteam`)?

Comment (vsoch) [reply docs/_docs/accounts/robots.md]: sure thing

**Comment by RonaldEnsing** [comment]
I think that something that is still missing from the documentation is that a 'last login' time needs to be specified as well. This can be done using the django administration by selecting the robot user and specifying a last login time before pressing 'save'.

**Comment by vsoch** [comment]
Would you care to PR to add this to your liking?

**Comment by vsoch** [comment]
Thanks again!

**Pull request merged**